### ISSUE TITLE
Compatibility with WP Google Maps Pro

### ISSUE THREAD

**Issue by DahmaniAdame** [wp-media/wp-rocket#4481]
**Before submitting an issue please check that you’ve completed the following steps:**
- [x] Made sure you’re on the latest version `3.10.1`
- [x] Used the search feature to ensure that the bug hasn’t been reported before

**Describe the bug**
When WP Google Maps Pro is used with Delay JavaScript Execution, the map is not displayed after user interaction and dependency errors are showing on the browser's console.

```
Uncaught TypeError: Cannot set properties of undefined (setting 'selector')
    at new a.fn.init (jquery-migrate.min.js:2)
    at n (jquery.js:2)
    at cart-fragments.min.js:1

wp-google-maps-pro.min.js?ver=8.1.18+pro-8.1.16:1 
        
       Uncaught TypeError: Cannot read properties of undefined (reading 'createInstance')
    at HTMLDocument.<anonymous> (wp-google-maps-pro.min.js:1)
    at n.fn.init.n.fn.ready.n.fn.init.ready (www.my-wildlife.com/:3)
    at a.fn.init.n.fn.init (jquery.js:2)
    at new a.fn.init (jquery-migrate.min.js:2)
    at n (jquery.js:2)
    at wp-google-maps-pro.min.js:1
    at HTMLDocument.<anonymous> (wp-google-maps-pro.min.js:1)
    at HTMLDocument.<anonymous> (www.my-wildlife.com/:3)
    at RocketLazyLoadScripts._triggerDOMContentLoaded (www.my-wildlife.com/:3)
    at async RocketLazyLoadScripts._loadEverythingNow (www.my-wildlife.com/:3)
```

Excluding the following fixes the issue: 

```
/jquery-?[0-9.](.*)(.min|.slim|.slim.min)?.js
/jquery-migrate(.min)?.js
/wp-google-maps(.*)
mgl_
wpgmza
```
**To Reproduce**
1. Use the WP Google Maps Pro plugin
2. Use Delay JavaScript Execution without any exclusions
3. Add a map
4. See the error

**Expected behavior**
The map should show up after user interaction. 

**Screenshots**
![image](https://user-images.githubusercontent.com/1289174/140310066-737c0b41-d5e4-4d4f-9203-abeb706e5e1e.png)

**Additional context**
Related - https://secure.helpscout.net/conversation/1684392233/305346/

**Backlog Grooming (for WP Media dev team use only)**
- [ ] Reproduce the problem
- [ ] Identify the root cause
- [ ] Scope a solution
- [ ] Estimate the effort

**Comment by piotrbak** [comment]
Will be fixed in the 3.13 backend

**Comment by piotrbak** [comment]
Handled in 3.13 using dynamic lists
